app/views/components/_error_summary.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if form.errors.any? %>
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      <% if form.errors.messages.size > 1 %>
+        Vyskytolo sa niekoľko problémov
+      <% else %>
+        Vyskytol sa problém
+      <% end %>
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <% form.errors.messages.each do |id, msg| %>
+          <li><a href="#apps_parliament_vote_app_application_form_<%= id %>" data-js-scroll-to><%= msg.first %></a></li>
+        <% end %>
+      </ul>
+    </div>
+  </div>
+<% end %>
+
+<script>
```

Comment:
Bez tohto po kliknuti na odkaz sa zmeni post na get a resetne to progress.

Reply:
To je dost divne nie? Preco sa to deje?

Reply:
Hmm ked ti mam pravdu povedat este som sa s tym nestretol a tiez mi to nedava zmysel.
Skusim este pogooglit ci nenajdem nieco.

![sd](https://user-images.githubusercontent.com/1857751/68621189-f708d700-0483-11ea-8810-9437436251f1.gif)

Reply:
Toto smrdi turbolinkami.

Reply:
Super postreh! Bolo to turbolinkami :)